ms-common/src/main/java/net/trajano/ms/core/Qualifiers.java

```diff
@@ -9,8 +11,11 @@
 
     /**
      * JWKS Cache Name.
+     * 
+     * @deprecated use {@link net.trajano.ms.spi.CacheNames#JWKS}
      */
-    public static final String JWKS_CACHE = "jwks_cache";
+    @Deprecated
+    public static final String JWKS_CACHE = CacheNames.JWKS;
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)